docs/extending/localtunneldebugger.md

```diff
@@ -0,0 +1,122 @@
+# Adding Providers to the Local Tunnel Debugger
+
+The Local Tunnel Debugger is intended to provide a local debugging option that includes
+the benefits of remote debugging. Traffic from the cluster is rerouted to the version of the application 
+running locally, so that developers can run and debug their code while freely communicating
+with resources in the cluster.
+
+Local Tunnel Debug providers can be installed via the marketplace. Developers of extensions with 
+tunnel debugging capabilities can register themselves as a Local Tunnel Debug provider in order
+to hook into the Kubernetes extension's API.
+
+## Elements of a debug provider
+
+An object that provides content for the Local Tunnel Debugger is called a _local tunnel debug provider_.
+Providers must be hosted within a Visual Studio Code extension.  This table summarises what
+your providers and their hosting extension need to do; the rest of this article goes into detail.
+
+| Component            | Responsibilities                                                     |
```

Comment:
💡  You can achieve this representation: 

![image](https://user-images.githubusercontent.com/6233295/104655510-ac878800-5722-11eb-9d86-1a0e19c38971.png)

From this code:

```
| Component            | Responsibilities                                                     |
|----------------------|----------------------------------------------------------------------|
| Your extension       | <ul><li>Activate when the Cluster Explorer is displayed or when the Debug (Local Tunnel) option is selected in the command palette</li><li>Register debug providers with Kubernetes extension</li><li>Tag itself as a local tunnel debug provider in the Visual Studio Marketplace</li></ul>        |
| Local Tunnel Debug provider| <ul><li>Implement the local tunnel debug provider interface</li><li>Allow users to configure which debug provider to use</li></ul>  |
| Kubernetes extension | Display the Debug (Local Tunnel) command in the palette or when interacting with supported resources in the Cluster Explorer                                 |
```

Reply:
Thanks so much! It looks so much better now :) I've updated the table items to be more descriptive as well.